.github/workflows/debian.yml

```
@@ -0,0 +1,90 @@
+# This action originally modeled off of the Debian build action:
+# https://salsa.debian.org/wouter/ola/-/blob/a38a396f6994b2b1af8efec9e208aee4e67f77aa/.gitlab-ci.yml
+
+name: debian
+on: [push, pull_request]
+jobs:
+  debian-build:
+    name: 'Debian Build ${{ matrix.image_tag }} amd64'
```

Given it seems to be trivially easy:
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Can we just hard-code a matrix of one (amd64) now, so that value can be used in the workflow name, image name etc.

```suggestion
    name: 'Debian Build ${{ matrix.image_tag }} ${{ matrix.architecture }}'
```

---

I'm going to push all of these as one commit instead of clicking accept on all of these individually.

---

Done.

---

> I'm going to push all of these as one commit instead of clicking accept on all of these individually.

You should also be able to queue them up and do them as one commit from GitHub's web interface if you want!